scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -223,40 +223,70 @@ def get_pet_subselect(fq_pet_table, samples, id):
 
 
 def populate_final_extract_table(fq_temp_table_dataset, fq_destination_table_data, fq_sample_mapping_table):
+  # first, create the table structure
   sql = f"""
         CREATE OR REPLACE TABLE `{fq_destination_table_data}` 
-        PARTITION BY RANGE_BUCKET(location, GENERATE_ARRAY(0, 26000000000000, 6500000000))
-        CLUSTER BY location
-        {FINAL_TABLE_TTL}
-        as (SELECT
-            new_pet.location,
-            s.sample_name as sample_name,
-            new_pet.state,
-            new_vet.ref,
-            REPLACE(new_vet.alt,",<NON_REF>","") alt,
-            new_vet.call_GT,
-            new_vet.call_GQ,
-            cast(SPLIT(new_vet.call_pl,",")[OFFSET(0)] as int64) as call_RGQ,
-            new_vet.QUALapprox,
-            new_vet.AS_QUALapprox,
-            new_vet.call_PL
-          FROM
-            `{fq_temp_table_dataset}.{PET_NEW_TABLE}` new_pet
-          LEFT OUTER JOIN
-            `{fq_temp_table_dataset}.{VET_NEW_TABLE}`  new_vet
-          ON (new_pet.location = new_vet.location AND new_pet.sample_id = new_vet.sample_id)
-          LEFT OUTER JOIN
-            `{fq_sample_mapping_table}` s ON (new_pet.sample_id = s.sample_id))
-      """
+        (
+              location      INT64,
+              sample_id	    INT64,
+              state	        STRING,
+              ref           STRING,
+              alt           STRING,
+              call_GT       STRING,
+              call_GQ       INT64,
+              call_RGQ      INT64,
+              QUALapprox    STRING,
+              AS_QUALapprox STRING,
+              call_PL       STRING	
+        )
+          PARTITION BY RANGE_BUCKET(location, GENERATE_ARRAY(0, 26000000000000, 6500000000))
+          CLUSTER BY location
+          {FINAL_TABLE_TTL}        
+        """
   print(sql)
-  existing_labels = client._default_query_job_config.labels
-  job_labels = existing_labels
-  job_labels["gvs_query_name"] = "populate-final-extract-table"
-  job_config = bigquery.QueryJobConfig(labels=job_labels)
-  cohort_extract_final_query_job = client.query(sql, job_config=job_config)
+  results = execute_with_retry("create-final-export-table", sql)
 
-  cohort_extract_final_query_job.result()
-  JOB_IDS.add((f"insert final cohort table {fq_destination_table_data}", cohort_extract_final_query_job.job_id))
+  sql = f"""
```

Comment:
is it more or less error prone in python to reuse the same variable name? `sql`
should we instead use `create_table_sql`, etc?